packages/synapse-react-client/src/components/EvaluationFinder/EvaluationFinder.test.tsx

```diff
@@ -46,7 +46,7 @@ describe('EvaluationFinder', () => {
     expect(evaluationsToSelect[1]).toBeChecked()
 
     const nextPageButton = await screen.findByRole('button', {
-      name: 'Next',
+      name: 'Next Page',
```

Comment:
When shown in the Submission View wizard, the modal also has a "Next" button. Adding "Page" here so we don't have two buttons with the same name